js/views/moderatorSettingsVw.js

```diff
@@ -40,7 +40,7 @@ module.exports = Backbone.View.extend({
       self.$el.html(loadedTemplate(self.model.toJSON()));
 
       //append the view to the passed in parent
-      self.parentEl.append(self.$el);
+      self.parentEl.show().append(self.$el);
```

Comment:
The bug is happening because the modal has a fadeOut when the transaction view closes, this should fadeIn to match it, like this:

self.parentEl.append(self.$el).fadeIn(300);